src/redux/actions/snackbar.js

```diff
@@ -0,0 +1,25 @@
+import { handleActions } from 'redux-actions';
+import actionTypes from '../actionTypes';
```

Comment:
Do need a separate file for this. It can be combined with the dialog box reducers.

Reply:
@akshatnitd The reason for keeping a separate file is, snackbar is used in more components, where dialog box is not used. Just to keep the imports less, so I split the reducers in to 2. If you want, I can combine them, or we can use 2 separate reducer

Reply:
I'll go with your approach, and make a single file